src/components/Explorer/Base/Result/RunLookup/RunLookup.tsx

```diff
@@ -32,10 +44,21 @@ export const RunLookup = ({ runId, filters }: Props) => {
     React.useEffect(() => {
         if (!runId) return
         getRunTitle(runId).then(setPageTitle)
+        callApi(runId).then((data) => {
+            setTableData(data)
+        })
```

Comment:
Remove unnecessary intermediate callback:

```suggestion
        callApi(runId).then(setTableData)
```

Reply:
Done: 476dd15